ROMFS/px4fmu_common/init.d/rc.uuv_apps

```diff
@@ -23,7 +23,8 @@ control_allocator start
 #
 # Start UUV Attitude Controller.
 #
-uuv_att_control start
+usv_att_control start
+usv_pos_control start
```

Comment:
This basically breaks all UUV use cases

Reply:
I will revert that commit. Should I create my own usv_apps? It is a bit similar to a boat and rover. So I don't know where I stand.

Reply:
Right, would it be too hard to genearlize the rover position controller for your use case? We can separate out the rover_att_control if you think is needed

Reply:
@Jaeyoung-Lim, I would appreciate a separate rover_pos_control, I think I might use it usv_att_control. I'm at loss with position control as I'm debugging attitude in on the real model. 

Still, should I add rc.usv_apps? I guess it would feature something like this: 
rover_pos_control
usv_att_control
6dof_rate_control – about this one I'm discussing at discord

Reply:
> rc.usv_apps?

Reply:
For now I would be running this module, but yes. rover should become more generic. My changes could be merged with it later.

Reply:
In that case, would you be able to work on separating the `rover_att_controller` outside of the rover position controller? Would be a nice addition if we can get https://github.com/PX4/PX4-Autopilot/pull/20082 merged together

Reply:
@Jaeyoung-Lim, would it be reasonable to drop this PR in favor of refactoring `rover_pos_control`? I think I gained enough know-how to extend it, and maybe add omni-directionality

Reply:
We can also keep this open, and rebase once the refactoring is done

Reply:
If your PR is in working order, I would prefer refactoring rover_pos_control to use control allocator